static/docs/command-reference/config.md

```diff
@@ -124,10 +124,10 @@ for more details.)
   effective of those two. DVC avoids `symlink` and `hardlink` types by default
   to protect user from accidental cache and repository corruption.
 
-  > **Note!** If you manually set `cache.type` to `hardlink` or `symlink`, **you
-  > will corrupt the cache** if you modify tracked data files in the workspace.
-  > See the `cache.protected` config option above and corresponding
-  > `dvc unprotect` command to modify files safely.
+  **Note** ⚠️ If you manually set `cache.type` to `hardlink` or `symlink`, **you
```

Comment:
Are you sure this a correct emoji here? Also, Would it be enough to just include emoji without **Note**?

Reply:
The concept of emoji correctness is very interesting indeed haha. You're right about removing "**Note**". Done that in 408073c1.